packages/cli/src/commands/generate/component/component.js

```diff
@@ -27,7 +27,7 @@ export const files = ({ name, typescript = false, ...options }) => {
     extension: `.stories${extension}`,
     webPathSection: REDWOOD_WEB_PATH_NAME,
     generator: 'component',
-    templatePath: 'stories.tsx.template',
+    templatePath: typescript ? 'stories.tsx.template' : 'stories.js.template',
```

Comment:
```suggestion
    // Using two different template files here because we have a TS-specific information in a comment in the .tsx template
    templatePath: typescript ? 'stories.tsx.template' : 'stories.js.template',
```